public/ifcWorker.js

```
@@ -104,11 +104,17 @@ self.onmessage = async (event) => {
         break;
 
       case "exportIfc":
+        // Handle the export data properly
+        const exportData = {
+          model: event.data.model,
+          fileName: event.data.fileName,
+          arrayBuffer: event.data.arrayBuffer
+        };
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)